staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/interface.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cel
+
+import (
+	v1 "k8s.io/api/admissionregistration/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/admission/plugin/webhook/generic"
+)
+
+// Validator is specific to validating admission policy
+type Validator interface {
+	// Validate is used to take cel evaluations and convert into decisions
+	Validate(evalResults []CELEvaluation, failPolicy *v1.FailurePolicyType) []PolicyDecision
```

Comment:
Move `CELEvaluation` into this file since the struct is part of the logical interface? Same for `CELValidation`.

Combined with removal of `CEL` prefixes, this should result in a tidy interface definition :)

Reply:
Yes thank you for the feedback! Done :)